README.md

```diff
@@ -48,7 +50,7 @@ If you have Jupyter installed, you can create a notebook using IRkernel from the
 You can also start other interfaces with an R kernel:
 
 ```bash
-# “ir” is the kernel name installed by the above 'IRkernel::installspec()'
```

Comment:
I don't know about R's unicode support, but in Python 2, using a non-ascii character in a source file without the encoding magic comment is a syntax error. In Python 3, it works so long as the file is saved in UTF-8, which may not be the default in all editors, especially on Windows. So there's clearly the potential for non-ascii characters to break stuff.